students/headieh_godwin/lesson05/mailroom.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python3
+import inquirer
+import os
+import time
+
+donor1 = {
+    "name": "Karen",
+    "donations": (20,20,100)
+}
+donor2 = {
+    "name": "Susan",
+    "donations": (20)
+}
+donor3 = {
+    "name": "Larry",
+    "donations": (40,50)
+}
+donor4 = {
+    "name": "Curly",
+    "donations": (20.99,20,100)
+}
+donor5 = {
+    "name": "Mo",
+    "donations": (2)
+}
+
+donors = {
+  "donor1": donor1,
+  "donor2": donor2,
+  "donor3": donor3,
+  "donor4": donor4,
+  "donor5": donor5
+}
+
+def menu():
+    """ Main function """
+    while True:
+        resp = original_prompt2()
+        if resp == '1':
+            thanks()
+        elif resp == '3':
+            thanks_all()
+        elif resp == '4':
+            print('bye')
+            break
+        elif resp == '2':
+            report()
+        else:
+            print('input is invalid')
+            break
+
+def original_prompt():
+    """ Prompt for a user selection """
+    questions = [
+        inquirer.List('action',
+                      message="What would you like to do? ",
+                      choices=['Send a Thank You', 'Create a Report', 'quit'])]
+    answers = inquirer.prompt(questions)
+    return(answers['action'])
+
+
+def original_prompt2():
+    answers = input(f"""
+Choose an action:
+
+1 - Send a Thank You to a single donor.
+2 - Create a Report.
+3 - Send letters to all donors.
+4 - Quit
+""")
+    return(answers)
+
+def email(x, y, z):
```

Comment:
use appropriate param names so that it is clear to the reader - x, y or z don't relay what their intent is?